src/svg-renderer.js

```diff
@@ -381,13 +389,23 @@ class SvgRenderer {
 
         const ratio = this.getDrawRatio() * (Number.isFinite(scale) ? scale : 1);
         const bbox = this._measurements;
-        this._canvas.width = bbox.width * ratio;
-        this._canvas.height = bbox.height * ratio;
+        
+        const scaledWidth = this._measurements.width * ratio;
+        const scaledHeight = this._measurements.height * ratio;
+
+        // Round render bounds to even integers to prevent non-integer rotation centers
+        this._renderBounds = {
+            width: Math.ceil(scaledWidth * 0.5) * 2,
+            height: Math.ceil(scaledHeight * 0.5) * 2,
+            x: this._measurements.x,
+            y: this._measurements.y
+        };
```

Comment:
Non-integer rotation centers should be OK... is this rounding necessary to fix LLK/scratch-render#233 or some other issue? It seems like this could introduce new problems with very small costumes if they happen to rasterize to an odd number of pixels.